### ISSUE TITLE
no attribute 'Equalize'

### ISSUE THREAD

**Issue by vadik6666** [albu/albumentations#351]
How should I use Equalize? 
I tried
```
import albumentations as A
print(A.__version__)
aug = A.Equalize()
```
and got `AttributeError: module 'albumentations' has no attribute 'Equalize'`

**Comment by Dipet** [comment]
Try install last version from git repo:
```
pip install git+https://github.com/albu/albumentations.git -no-cache-dir
```

**Comment by vadik6666** [comment]
I used `'0.3.2'`

**Comment by Dipet** [comment]
Version from PyPi does not contains this transform. You need get latest version from GitHub
Use this code for upgrade your current version
```
pip install -U git+https://github.com/albu/albumentations
```
Or this code for install latest version from GitHub:
```
pip install git+https://github.com/albu/albumentations.git --no-cache-dir
```

**Comment by albu** [comment]
fixed in 0.3.3

**Comment by vadik6666** [comment]
@Dipet , @albu thank you!
